src/browser/api/window.js

```diff
@@ -121,9 +121,9 @@ let browserWindowEventMap = {
 };
 
 let webContentsEventMap = {
-    'did-get-response-details': {
-        topic: 'resource-response-received',
```

Comment:
`resource-response-received` is actually is an outward facing event that has a specific shape and is documented [here](http://cdn.openfin.co/jsdocs/stable/tutorial-window.addEventListener.html). A lot of this shouldn't be changed.